src/app/_components/Icon.tsx

```diff
@@ -5,25 +5,31 @@ export type IconProps = {
   color?: keyof typeof colorStyles
   size?: number
   weight?: 'light' | 'regular' | 'bold'
+  inheritHoverStyle?: boolean
```

Comment:
Are there instances where we don't want this to happen?

Reply:
This would have impact on some other parts of the UI, thought it was safer if we explicitly pass a prop

https://github.com/user-attachments/assets/f335f77c-72cf-4a0f-b114-1953deb3b0c7